.gitignore

```diff
@@ -1,3 +1,5 @@
 target
 local-*
-dependency-reduced-pom.xml
\ No newline at end of file
+dependency-reduced-pom.xml
+.idea
+*.iml
```

Comment:
I added ignores for IntelliJ project files so I didn't accidentally check them in.